de.json

```diff
@@ -980,10 +980,10 @@
 			"tooltip-clear-history": "Suchverlauf löschen",
 			"label-path-option-description": "Pfad stimmt überein",
 			"label-file-name-option-description": "Dateiname stimmt überein",
-			"label-tag-option-description": "suche nach Tags",
+			"label-tag-option-description": "Tag stimmt überein",
 			"label-line-option-description": "suche Begriff in der gleichen Zeile",
 			"label-section-option-description": "suche Begriff unter der gleichen Überschrift",
-			"label-property-option-description": "match property",
+			"label-property-option-description": "Eigenschaften stimmt überein",
```

Comment:
"Eigenschaft stimmt überein"

Reply:
Hi @joethei, could you be a bit more specific? Do you see a mistake in the translation?

Reply:
The English original is using singular, while your translation is using plural.
(and only partially at that)

Reply:
Danke. Hatte Tomaten auf den Augen. Habe es entsprechend korrigiert.